extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -70,3 +70,8 @@ amp-ad[data-a4a-upgrade-type="amp-ad-network-adsense-impl"] > iframe {
   min-width: 0;
 }
 
+amp-ad[data-a4a-upgrade-type="amp-ad-network-doubleclick-impl"][height="fluid"] > iframe {
```

Comment:
FYI: I toyed with moving this to a separate CSS file for amp-ad-network-doubleclick-impl, but it seems like those styles get applied later than those in amp-ad.css, so much so that it doesn't work the same if I do it that way.